lib/daru/vector.rb

```diff
@@ -879,41 +879,56 @@ def to_s
       to_html
     end
 
-    # Create a summary of the Vector using Report Builder.
-    def summary(method=:to_text)
-      ReportBuilder.new(no_title: true).add(self).send(method)
-    end
-
-    # :nocov:
-    def report_building b # rubocop:disable Metrics/AbcSize,Metrics/MethodLength
-      b.section(name: name) do |s|
-        s.text "n :#{size}"
-        s.text "n valid:#{count_values(*Daru::MISSING_VALUES)}"
-        if @type == :object
-          s.text  "factors: #{factors.to_a.join(',')}"
-          s.text  "mode: #{mode}"
-
-          s.table(name: 'Distribution') do |t|
-            frequencies.sort_by(&:to_s).each do |k,v|
-              key = @index.include?(k) ? @index[k] : k
-              t.row [key, v, ('%0.2f%%' % (v.quo(count_values(*Daru::MISSING_VALUES))*100))]
-            end
-          end
+    # Create a summary of the Vector
+    def summary(indent_level=0) # rubocop:disable Metrics/AbcSize,Metrics/MethodLength
+      summary = '  =' * indent_level + "= #{name}"
+      summary << "\n  n :#{size}"
+      summary << "\n  n valid:#{count_values(*Daru::MISSING_VALUES)}"
+      if @type == :object
+        summary << "\n  factors: #{factors.to_a.join(',')}"
+        summary << "\n  mode: #{mode}"
+
+        # Display in table format
+        summary << '  ' * indent_level + '  Distribution'
+        key = []
+        val = []
+        nval = []
+        frequencies.sort_by(&:to_s).each do |k,v|
+          key << (@index.include?(k) ? @index[k] : k)
+          val << v
+          nval << ('%0.2f%%' % (v.quo(count_values(*Daru::MISSING_VALUES))*100))
         end
 
-        s.text "median: #{median}" if @type==:numeric || @type==:numeric
-        if @type==:numeric
-          s.text 'mean: %0.4f' % mean
-          if sd
-            s.text 'std.dev.: %0.4f' % sd
-            s.text 'std.err.: %0.4f' % se
-            s.text 'skew: %0.4f' % skew
-            s.text 'kurtosis: %0.4f' % kurtosis
-          end
+        summary << table(key, val, nval)
+      end
+
+      if @type == :numeric
```

Comment:
Why not `elsif`? Vector can't be object and numeric at the same time. Or, in fact, `case @type` would be more reasonable.